cmd/openshift-tuned.go

```diff
@@ -253,9 +299,15 @@ func mainLoop(clientset *kubernetes.Clientset, nodeName string, resyncPeriodDura
 	ticker := time.NewTicker(time.Second * time.Duration(resyncPeriodDuration))
 	go func() {
 		for range ticker.C {
+			// Compare node labels
 			nodeLabels := nodeLabelsGet(clientset, nodeName)
-			nodeLabelsCompare(nodeLabelsOld, nodeLabels)
+			labelsCompare(nodeLabelsOld, nodeLabels, "node", fileNodeLabels)
```

Comment:
If `fileNodeLabels` is a global variable you don't need to pass it around, maybe it doesn't need to be global?